### PR TITLE
[Ag Grid Theme] Set tooltip background color

### PR DESCRIPTION
**Pull request by alycrys**
Closes https://github.com/jpmorganchase/salt-ds/issues/2651

### CODE REVIEW

File: packages/ag-grid-theme/css/_salt-ag-theme-default-params.scss [hunk 1]
```diff
@@ -3,6 +3,7 @@
 $salt-ag-theme-default-params: (
   row-hover-color: var(--agGrid-row-color-hover),
   row-border-color: var(--agGrid-row-borderColor),
+  header-background-color: var(--salt-color-white),
```
Comment (origami-z) [inline packages/ag-grid-theme/css/_salt-ag-theme-default-params.scss:6]: what about dark mode?

**Review by origami-z — COMMENTED** [review]
needs changeset to release

**Review by origami-z — APPROVED** [review]
Tested in https://stackblitz.com/edit/salt-ag-grid-theme-h7vsly?file=package.json,App.jsx

Would be good to have an example / snapshot in the next branch.

**Comment by alycrys** [comment]
> Tested in https://stackblitz.com/edit/salt-ag-grid-theme-h7vsly?file=package.json,App.jsx
> 
> Would be good to have an example / snapshot in the next branch.

yep, just about to push it

**Review by feliciackh — APPROVED** [review]

**Pull request merged**